tests/console/libssh.pm

```diff
@@ -0,0 +1,151 @@
+    create_image if (script_output('docker images') !~ m/libssh_image/);    #Skip building If image is already available. This is for Fips test
+    assert_script_run("iptables -I INPUT 5 -p tcp --dport 22 -j ACCEPT");
+    assert_script_run("mkdir /tmp/test; echo -n libssh_testcase001 > /tmp/test/libssh_testfile");
+    assert_script_run("qemu-img create -f raw /tmp/test/libssh_block.raw 20m");    #file to be uploaded to container for qemu block test
+    assert_script_run('docker run -itd --name libssh_container --hostname container01 --add-host susetest:$(ip -4 addr show docker0 | grep -Po "inet \K[\d.]+") --privileged libssh_image');
+
+    #Create key pair on host and container,  import sshd host key from container, copy pub key to each other
+    my $container_ip = script_output("docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' libssh_container");
+    assert_script_run("ssh-keygen -t rsa -N '' -m pem -f /root/.ssh/id_rsa <<< y");    #Current libssh2 only support pem file format
+    assert_script_run("ssh-keyscan $container_ip >> /root/.ssh/known_hosts");
+
+    assert_script_run("docker exec libssh_container ssh-keygen -t rsa -N '' -m pem -f /root/.ssh/id_rsa <<< y");
+    assert_script_run("docker cp /root/.ssh/id_rsa.pub libssh_container:/root/.ssh/authorized_keys");
+    assert_script_run("docker cp libssh_container:/root/.ssh/id_rsa.pub /root/.ssh/authorized_keys");
+
+    #Switch into container as client
+    type_string("docker exec -it libssh_container bash\n", wait_still_screen => 3);
+    assert_script_run("test -f /.dockerenv");                                          #verify inside container
+    assert_script_run("ssh-keyscan susetest >> /root/.ssh/known_hosts");
+    validate_script_output("curl -s sftp://susetest/tmp/test/ -u root:nots3cr3t",                sub { m/libssh_testfile/ });
+    validate_script_output("curl -s sftp://susetest/tmp/test/libssh_testfile -u root:nots3cr3t", sub { m/libssh_testcase001/ });
+    assert_script_run("curl -s sftp://susetest/tmp/test/libssh_block.raw -u root:nots3cr3t -o /tmp/libssh_block.raw");
+    validate_script_output("curl -s scp://susetest/tmp/test/libssh_testfile -u root:nots3cr3t",             sub { m/libssh_testcase001/ });
+    validate_script_output('curl -s sftp://root@susetest/tmp/test/libssh_testfile --key /root/.ssh/id_rsa', sub { m/libssh_testcase001/ });
+    validate_script_output('curl -s scp://root@susetest/tmp/test/libssh_testfile --key /root/.ssh/id_rsa',  sub { m/libssh_testcase001/ });
+    validate_script_output('virsh -c "qemu+libssh://root@susetest/system?sshauth=privkey&keyfile=/root/.ssh/id_rsa&known_hosts=/root/.ssh/known_hosts" hostname', sub { m/susetest/ }) if is_sle('>=15'); #libssh is not supported by libvirt for sle12
+    validate_script_output('virsh -c "qemu+libssh2://root@susetest/system?sshauth=privkey&keyfile=/root/.ssh/id_rsa&known_hosts=/root/.ssh/known_hosts" hostname', sub { m/susetest/ });
```

Comment:
```suggestion
    if is_sle('>=15') { #libssh is not supported by libvirt for sle12
        validate_script_output('virsh -c "qemu+libssh://root@susetest/system?sshauth=privkey&keyfile=/root/.ssh/id_rsa&known_hosts=/root/.ssh/known_hosts" hostname', sub { m/susetest/ });
    } else {
        validate_script_output('virsh -c "qemu+libssh2://root@susetest/system?sshauth=privkey&keyfile=/root/.ssh/id_rsa&known_hosts=/root/.ssh/known_hosts" hostname', sub { m/susetest/ });
    }
```

Reply:
The original logic will change:
original logic : "virsh -c "qemu+libssh2" will run for all sle versions.
After change as you suggested: "virsh -c "qemu+libssh2" will not run if is_sle('>=15')

Reply:
OK, now I see the difference. Sorry